src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+:root {
+  --link-color: #00acdc;
+  --color: #616070;
+  --bg-color: #fff;
+  --test-color: #060b35;
```

Comment:
The variable name `--test-color` does not clearly indicate its purpose. Consider renaming it to reflect its usage context, following the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) for naming conventions.

Reply:
done